common/app/common/NewNavigation.scala

```diff
@@ -535,9 +366,67 @@ object NewNavigation {
     )
 
     def getTertiaryNavLinks(frontId: String): List[TertiaryLink] = {
-      tertiaryLinks.filter{ item =>
+      tertiaryLinks.filter { item =>
         item.frontId == frontId
       }
     }
   }
+
+  object SectionLinks {
+
+    var sectionLinks = List(
+
+      SectionsLink("uk", News),
```

Comment:
Yes I agree! Certainly if this change is kept I want to sit down and discuss a nice way to avoid repetition, but still make it feel not so complicated.